### PR TITLE
Add PHP Core Roundup #10

### PR DESCRIPTION
**Pull request by Ayesh**

### CODE REVIEW

File: source/_posts/2023-03-01-php-core-roundup-10.md [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+Welcome back to the tenth in the PHP Core Roundup series. As February comes to end we have plenty of PHP developments to write about. February was a month full of new RFCs for upcoming PHP 8.3, merged changes, security releases for PHP 8.2, 8.1, and 8.0, as well as several discussions in the mailing list about improving PHP. 
+
+> The PHP Foundation currently supports six part-time PHP contributors who work on maintenance and new features for PHP. Maintenance is not limited to fixing bugs, but also includes work to reduce technical debt, making life easier for everyone working on PHP. The contributors funded by the PHP Foundation collaborate with other contributors on code, documentation, and discussions.
+
+<br>
+We publish the posts on our website, and you can subscribe to a newsletter; You don’t necessarily have to be a PHP Foundation backer to follow _PHP Core Roundup_.
+
+{% include "newsletter.html" %}
+
+## Call for PHP 8.3 Release Managers
```
Comment (saundefined) [inline source/_posts/2023-03-01-php-core-roundup-10.md:22]: Today we published the deadline for applications and voting phase:
- Applications will be accepted until 31 March 12:00:00 UTC
- Elections (if needed) will start on 1 April and run until 16 April 12:00:00 UTC

You can add this if you like :)

Comment (Ayesh) [reply source/_posts/2023-03-01-php-core-roundup-10.md:22]: Awesome thank you @saundefined. I added these dates to the post.

File: source/_posts/2023-03-01-php-core-roundup-10.md [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+- [RFC Idea - json_validate() validate schema](https://externals.io/message/119631)
+
+
+## Merged PRs and Commits
+
+Following are some changes that did not go through an RFC process because they are either planned, bug fixes, or progressive enhancements. There are automated unit and integration tests for each of these changes, the PHP core developers review all pull requests.
+
+### Change Highlights
+ 
+ - Max Kellermann continued to clean-up and improve several extensions and the Zend engine.
```
Comment (pronskiy) [inline source/_posts/2023-03-01-php-core-roundup-10.md:109]: It looks like it's a controversial change, and not everyone agrees it actually improves things. Remove at all? Or remove "and improve".

File: source/_posts/2023-03-01-php-core-roundup-10.md [hunk 3]
```diff
@@ -0,0 +1,320 @@
...
+
+
+ 
+### Notable Mailing List Discussions
+
+- [Official Preprocessor](https://externals.io/message/119451)
+- [Deprecate ldap_connect with host and port as separate arguments](https://externals.io/message/119425)
+- [RFC proposal: function array_filter_list() to avoid subtle bugs/workarounds](https://externals.io/message/119444)
+- [RFC Proposal - Types for Inline Variables](https://externals.io/message/119470)
+- [Windows PECL build machine died](https://externals.io/message/117494)
```
Comment (pronskiy) [inline source/_posts/2023-03-01-php-core-roundup-10.md:97]: This one is old, remove?

File: source/_posts/2023-03-01-php-core-roundup-10.md (hunk 2 above)
Comment (Ayesh) [reply source/_posts/2023-03-01-php-core-roundup-10.md:109]: Yeah those recent emails were pretty controversial. I will leave to "clean-up" for now. Thank you.

File: source/_posts/2023-03-01-php-core-roundup-10.md (hunk 3 above)
Comment (Ayesh) [reply source/_posts/2023-03-01-php-core-roundup-10.md]: You are right, removing.

**Pull request merged**